### PR TITLE
Fix psalm upload coverage

### PR DESCRIPTION
**Pull request by terabytesoftw**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  | https://github.com/yiisoft/yii-dataview/issues/50

### CODE REVIEW

File: src/BaseListView.php [hunk 1]
```diff
@@ -30,21 +29,21 @@ abstract class BaseListView extends Widget
     public const BOOTSTRAP = 'bootstrap';
     public const BULMA = 'bulma';
     protected array $options = [];
-    protected PaginatorInterface $paginator;
+    protected OffsetPaginator $paginator;
```
[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/BaseListView.php:32]: Example ListView with paginator [https://test.mister42.eu/articles/](url)

[the remark this answers is not in the document]
Comment (samdark) [reply src/BaseListView.php:32]: I mean keyset paginator.

[the remark this answers is not in the document]
Comment (terabytesoftw) [reply src/BaseListView.php:32]: For now the module has been cleaned, the tests have been added, I want it to work well first, then I will think about how to add KeysetPaginator, if I use the interface I will have to add many @ psalm-suppress thanks to the fact that OffsetPaginator has methods obviously that KeysetPaginator does not have , and I can't add them to the common interface.

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
👍

**Pull request merged**